DigitalLearningSolutions.Data/Services/TutorialContentService.cs

```diff
@@ -176,7 +178,8 @@ FROM CustomisationTutorials
                          AND Sections.SectionID = @sectionId
                          AND Tutorials.TutorialId = @tutorialId
                          AND Customisations.Active = 1
-                         AND CustomisationTutorials.Status = 1;",
+                         AND CustomisationTutorials.Status = 1
+                         AND Tutorials.ArchivedDate IS NULL;",

```

Comment:
Are we checking the section isn't archived?

Reply:
No, added this change in #154 🙂